testing/Cargo.toml

```diff
@@ -7,8 +7,8 @@ authors = ["Victor Koenders <vkoenders@espressosys.com>"]
 
 [features]
 default = []
-full-ci = ["async-std-executor", "demo"]
-tokio-ci = ["tokio-executor", "demo"]
+full-ci = ["async-std-executor", "demo", "channel-async-std"]
```

Comment:
Just confirming, this is the crate that refers to, rather than being included in, the top level crate, correct?

Reply:
Yes!